srcpkgs/PackageKit-qt/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'PackageKit-qt'
+pkgname=PackageKit-qt
+version=1.1.0
+revision=1
+archs="x86_64"
+#wrksrc=
+#create_wrksrc=yes
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="PackageKit"
```

Comment:
this should probably be PackageKit-devel and in makedepends

Reply:
I get an xlint error if I have nothing in the hostmakedepends. How do I resolve this?

Reply:
if the variable is empty you don't need to include it in the template